.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -30,4 +44,13 @@ see examples:
 
 - https://checkstyle.org/report_issue.html#How_to_request_new_Check.2FModule.3F
 
+- https://github.com/checkstyle/checkstyle/issues/8764
```

Comment:
please move it inside "report_issue.html"

Reply:
Done